sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.createOffering.e2e-spec.ts

```diff
@@ -197,6 +198,67 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-createOffering", (
     );
   });
 
+  it("Should throw error when education program is expired.", async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
```

Comment:
Even having cache expiration in place, please acquire the token as close as possible to the API request, as it is usually built in other E2E tests where the endpoint and token are the last part before the endpoint call.